workflow-management/actions.mdx

```diff
@@ -22,7 +22,7 @@ These actions help control the sequence of operations in your workflow and make
 
 * **Condition**: split the workflow into exactly two branches based on a [condition](/workflow-management/conditional-branching/).
 * **Split**: split the workflow into multiple branches based on a [condition](/workflow-management/conditional-branching/).
-* **Loop on Items**: iterate over a list of items obtained in a prior step. For better performance, OpenOps enables parallel execution of loop iterations whenever possible.
+* **Loop on Items**: Iterate over a list of items obtained in a prior step, executing each iteration sequentially. All iterations run to completion regardless of failures or pauses in individual iterations. After all iterations finish, the loop’s final state reflects the outcomes of all iterations: if any iteration fails, the loop step fails; if any iteration pauses, the loop step pauses.
```

Comment:
Not sure I understand. If an iteration pauses and execution is sequential, how is it possible to run the entire loop to completion? If a loop is complete, doesn't it mean that no iterations are paused?

Also, a bit of nitpicking: the first word following the colon starts with a lowercase latter: `**Loop on Items**: iterate`